doc/src/build/programming-with-objects/ch1-object-basics.md

```diff
@@ -0,0 +1,111 @@
+To make an object owned by an account address, the API we need to call is defined in the Transfer library:
+```
+public fun transfer<T: key>(obj: T, recipient: address)
+```
+A typical use case is to transfer the object to the sender/signer of the current transaction. The only way to obtain the sender of the current transaction, is to rely on the transaction context passed in from an entry function. The last argument to an entry function must be the current transaction context, defined as `ctx: &mut TxContext`.
+To obtain the current signer's address, one can call `TxContext::sender(ctx)`.
+Below is the code that creates a new `ColorObject` and make it owned by the sender of the transaction:
+```
+use Sui::Transfer;
+
+// This is an entry function that can be called directly by a Transaction.
+public fun create_color_object(red: u8, green: u8, blue: u8, ctx: &mut TxContext) {
+    let color_object = new_color_object(red, green, blue, ctx);
+    Transfer::transfer(color_object, TxContext::sender(ctx))
+}
+```
+
+You can find the full code [here](../../../move_code/objects_tutorial/sources/Ch1.move).
+
+### Onchain Interactions
+Now let's try to call the `create_color_object` in transactions and see what happens. To do this we need to start Sui and the wallet. Please follow the [Wallet guide](../wallet.md) to start the Sui network and setup the wallet.
+
+Before starting, let's take a look at the default wallet address (this will be the address that will eventually own the object latter):
+```
+wallet --no-shell active-address
+```
+It will tell you the current wallet address.
+
+First of all, we need to publish the code onchain. Assuming the path to the root of the repository is $ROOT:
+```
+wallet --no-shell publish --path $ROOT/doc/move_code/objects_tutorial --gas-budget 10000
+```
+You can find the published package object ID in the **Publish Results**, like this:
+```
+----- Publish Results ----
+The newly published package object: (57258F32746FD1443F2A077C0C6EC03282087C19, SequenceNumber(1), o#b3a8e284dea7482891768e166e4cd16f9749e0fa90eeb0834189016c42327401)
+```
+Note that the exact data you see will be different. The first hex string in that triple is the package object ID (57258F32746FD1443F2A077C0C6EC03282087C19 in this case).
+Next we can call the function to create a color object:
+```
+wallet --no-shell call --gas-budget 1000 --package "57258F32746FD1443F2A077C0C6EC03282087C19" --module "Ch1" --function "create_color_object" --args 0 255 0
+```
+In the **Transaction Effects**, you will see an object showing up in the list of **Created Objects**, like this:
+```
+Created Objects:
+5EB2C3E55693282FAA7F5B07CE1C4803E6FDC1BB SequenceNumber(1) o#691b417670979c6c192bdfd643630a125961c71c841a6c7d973cf9429c792efa
+```
+We can inspect this object and see what kind of object it is:
+```
+wallet --no-shell object --id 5EB2C3E55693282FAA7F5B07CE1C4803E6FDC1BB
```

Comment:
It might be cool to pass`--json` here so they can see the structure of the object, or to show that in a follow-up command to explain the difference between object metadata and type-specific data.